HW13/task1.py

```diff
@@ -0,0 +1,82 @@
+import json
+
+phone_book = {}
+
+# Завантаження даних з файлу
+try:
+    with open('phone_book.json', 'r') as f:
+        phone_book = json.load(f)
+except FileNotFoundError:
+    phone_book = {}
+
+#Збереження даних у файл
+def save_data():
```

Comment:
nit: better to use `phone_book` as a parameter
```suggestion
def save_data(phone_book):
```